lib/Button/Button.less

```diff
@@ -16,6 +27,14 @@
 	padding: 0 @moon-button-h-padding;
 	margin: 0 @moon-spotlight-outset;
 	color: @moon-button-text-color;
+	-webkit-transform: translateZ(0);
+	-webkit-transition: transform 0.2s ease-out;
+	transform: translateZ(0);
+	transition: transform 0.2s ease-out;
+	-webkit-animation-duration: 0.2s;
```

Comment:
We can't (we can, but it's not supported yet). keyframe animation isn't done yet. Everyone still uses prefixes. (This surprised me so I specifically looked into it before submitting this PR.)